znoyder/browser.py

```diff
@@ -72,7 +74,10 @@ def get_packages(**kwargs):
                     if kwargs.get('upstream') in str(package.get('upstream'))]
 
     for package in packages:
-        package['osp-project'] = urlparse(package['osp-patches']).path[1:]
+        if package['osp-patches']:
+            repo_name = re.search(r'^(.*)\/(.*)$', package['osp-patches']).group(2)
```

Comment:
```
pep8: commands[0]> flake8
./znoyder/browser.py:78:80: E501 line too long (83 > 79 characters)
```

Reply:
why it is under conditional? This may change the original behavior – the osp-project shall be always defined, based on osp-patches, so if we have no osp-patches we probably have some issues in ospinfo...

Reply:
Fixed directly here to move forward with blocked work